app/views/devise/confirmations/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
```

Comment:
I think we need to use `slim`

Reply:
Changed all `erb` formt files to `slim`.